tests/src/model/pages/images-page.ts

```diff
@@ -57,6 +80,13 @@ export class ImagesPage extends MainPage {
     return new ImageDetailsPage(this.page, name);
   }
 
+  async pruneImages(): Promise<ImagesPage> {
+    await this.pruneImagesButton.click();
+    await this.pruneConfirmationButton.waitFor({ state: 'visible', timeout: 3000 });
+    await this.pruneConfirmationButton.click();
```

Comment:
same as above-we can use handleconfirmationdialog function from operations utility file.